navigation/OnboardingStack.tsx

```diff
@@ -1,19 +1,28 @@
-import React from 'react';
-import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import OnboardingScreen from '../screen/wallets/OnboardingScreen';
+import React from "react";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import OnboardingScreen from "../screen/wallets/OnboardingScreen";
+import CreateRecoveryPhraseScreen from "../screen/wallets/CreateRecoveryPhraseScreen";
+import PleaseBackup from "../screen/wallets/PleaseBackup";
+import { RouteProp } from '@react-navigation/native';
 
 export type OnboardingStackParamList = {
-  OnboardingMain: undefined;
-};
+    OnboardingMain: undefined;
+    CreateRecoveryPhrase: { walletID: string };
```

Comment:
Once we move the wallet creation logic to `CreateWallet`, walletId would no longer be required